### DISCUSSION TITLE
mac版maa是不是没有win版maa的连战次数？

### DISCUSSION THREAD

**Discussion by lizrunn** [MaaAssistantArknights/MaaAssistantArknights#9205] [Q&A]
win版maa可以通过设置“连战次数”，在明日方舟中选择一次性战斗的次数。
![屏幕截图 2024-05-25 231908](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/105421651/e404f2ea-db0e-4a0c-83aa-11668cd4d4c7)

mac版maa有一个“指定次数”，指的是单独作战几次，和win版maa的“连战次数”并不一样。
<img width="427" alt="image" src="https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/105421651/ab2a88fb-ff7c-4559-8078-159e2b46dbe7">

可不可以在mac版maa上也加个“连战次数”？

**Comment by ABA2396** [comment]
cc @hguandl

**Comment by ABA2396** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/9206
